src/tests/testHttpRequest.cc

```diff
@@ -45,60 +45,59 @@ testHttpRequest::testCreateFromUrl()
 {
     /* vanilla url, implict method */
     unsigned short expected_port;
-    char * url = xstrdup("http://foo:90/bar");
+    const char * url = "http://foo:90/bar";
     const MasterXaction::Pointer mx = new MasterXaction(XactionInitiator::initClient);
-    HttpRequest *aRequest = HttpRequest::FromUrl(url, mx);
+    auto aRequest = HttpRequest::FromUrlXXX(url, mx);
```

Comment:
I still think that most if not all of these test cases should use FromUrl(). You are already modifying the lines initializing these URLs. The only other modification would be the last CPPUNIT_ASSERT_EQUAL() in the blob that should be removed as no longer relevant anyway!

Reply:
GitHub is having a hard time grouping review comments for this old PR with many force-pushes so I am adding this comment to reduce the chance of this older change request getting lost in the noise.